### DISCUSSION TITLE
Stellar (XLM) assets and trustlines

### DISCUSSION THREAD

**Discussion by tskaggs** [LedgerHQ/ledger-live-desktop#4225] [Ideas]
Hi All! 👋

I saw an "issue" submitted as a request to show and calculate Stellar(XLM) assets and trustlines. Is this in the works? If not, would a contribution be cool with y'all?

Resources:
* [Original "issue"](https://github.com/LedgerHQ/ledger-live-desktop/issues/3921)
* [Stellar Assets](https://developers.stellar.org/docs/glossary/assets/)
* [A Guide to Trustlines on Stellar](https://medium.com/stellar-community/a-guide-to-trustlines-on-stellar-8bc46091a86f)

**Comment by Stumblinbear** [comment]
I'm looking into developing this myself, though I'm unsure of the best practices (or, how it would best be implemented) for this repo.
